.travis.yml

```diff
@@ -9,15 +10,20 @@ env:
 - BUILDER_NAME="ovs-cni/${TRAVIS_JOB_ID}builder"
 
 install: true
+script:
+  - cd ../..
+  - mv kubevirt ../kubevirt.io
+  - cd ../kubevirt.io/ovs-cni
+  - make docker-build
```

Comment:
Don't you need to pass TAG variable when building?

You either can move this line to deploy section or better, run `docker tag` in the deploy section.

Reply:
Good one!